zephyr/module.yml

```diff
@@ -61,52 +77,52 @@ blobs:
+    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libcore.a
     description: "Binary libraries supporting the ESP32 series RF subsystems"
     doc-url: https://github.com/espressif/esp32-wifi-lib
-  - path: lib/esp32/libmesh.a
-    sha256: 171582f2a01f3c6fe403e58e217114c46fe9b5aecc4caa3a31b5c5349ebcf85c
+  - path: lib/esp32/libespnow.a
+    sha256: 2e8d91b1e74b7ea6a3a93bdb4e4ba673310bb84d7da03b2d2aa132186784e38a
     type: lib
     version: '1.0'
     license-path: zephyr/blobs/license.txt
-    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libmesh.a
+    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libespnow.a
     description: "Binary libraries supporting the ESP32 series RF subsystems"
     doc-url: https://github.com/espressif/esp32-wifi-lib
-  - path: lib/esp32/libwapi.a
-    sha256: 95af3a25fa36044b58db121809cd4a759636faca05c4f57308c176d83e1c62ed
+  - path: lib/esp32/libnet80211.a
+    sha256: c2433ae08915d1bebf2b2162f0e16edb7e9f2568a9333aa244d70482d2d32600
     type: lib
     version: '1.0'
     license-path: zephyr/blobs/license.txt
-    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libwapi.a
+    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libnet80211.a
     description: "Binary libraries supporting the ESP32 series RF subsystems"
     doc-url: https://github.com/espressif/esp32-wifi-lib
-  - path: lib/esp32/libespnow.a
-    sha256: 2e8d91b1e74b7ea6a3a93bdb4e4ba673310bb84d7da03b2d2aa132186784e38a
+  - path: lib/esp32/libwapi.a
+    sha256: 95af3a25fa36044b58db121809cd4a759636faca05c4f57308c176d83e1c62ed
     type: lib
     version: '1.0'
     license-path: zephyr/blobs/license.txt
-    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libespnow.a
+    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libwapi.a
     description: "Binary libraries supporting the ESP32 series RF subsystems"
     doc-url: https://github.com/espressif/esp32-wifi-lib
-  - path: lib/esp32/libnet80211.a
-    sha256: c2433ae08915d1bebf2b2162f0e16edb7e9f2568a9333aa244d70482d2d32600
+  - path: lib/esp32/libsmartconfig.a
+    sha256: 48a5cda7c5c7bfab9e3368d041e44f552a904a48ed53fc32d00e8a61d3ba6a42
     type: lib
     version: '1.0'
     license-path: zephyr/blobs/license.txt
-    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libnet80211.a
+    url: https://github.com/espressif/esp32-wifi-lib/raw/52bbf235d23ef4c647cf95abe4952a260c4d0662/esp32/libsmartconfig.a
     description: "Binary libraries supporting the ESP32 series RF subsystems"
     doc-url: https://github.com/espressif/esp32-wifi-lib
-  - path: lib/esp32/libcore.a
```

Comment:
I was about to write that the non-esp32c2 entries should be the same as it was not updated. Now I see that it was only moved around due to `esp_genblobs.py` changes.